rpc/manager/manager.proto

```diff
@@ -429,6 +429,8 @@ message License {
   // cluster_id is really the id of the namespace where the traffic-manager is installed.
   string cluster_id = 3;
   string err_msg = 4;
+  // is_license_valid indicates whether the license is valid.
+  bool is_license_valid = 5;
```

Comment:
```suggestion
  bool valid = 5;
```
This is a field in the structure named "License" so repeating the word "license" in the field name is redundant. Also, since it's a `bool`, it either is or isn't, which makes the "is" word redundant too.

Based on that, I think the name "valid" is better. This in turn makes the comment "valid indicates whether the license is valid" somewhat redundant too.

Reply:
Thanks @thallgren, I renamed it to `valid`